src/components/inputs/select-input/select-input.spec.js

```diff
@@ -181,6 +181,22 @@ describe('in multi mode', () => {
     });
   });
   describe('interacting', () => {
+    describe('when disabled', () => {
+      it('should call not onChange when value is cleared', () => {
+        const onChange = jest.fn();
+        const { getByLabelText, queryByText } = renderInput({
+          onChange,
+          isMulti: true,
+          value: ['mango'],
+          isDisabled: true,
+        });
+        const input = getByLabelText('Fruit');
+        fireEvent.focus(input);
+        fireEvent.keyDown(input, { key: 'Backspace' });
+        expect(onChange).not.toHaveBeenCalled();
+        expect(queryByText('Mango')).toBeInTheDocument();
+      });
+    });
```

Comment:
I didn't add a test for clicking to delete because of [this](https://github.com/testing-library/react-testing-library/issues/356)